src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./Footer.module.css";
+import { Link } from "react-router-dom";
+import Logo from "../../assets/AegisStore.svg";
+
+export default function () {
+  return (
+    <footer className={`shadow ${styles.footer}`}>
+      <div className="main-container">
+        <div className={styles.footer__container}>
+          <div className="logo-container">
+            <img src={Logo} alt="logo" />
+          </div>
+          <div className="links-container">
+            <div className={styles.links__heading}>Quick Links</div>
+            <ul className={`${styles.list}`}>
+              <li className={styles.list__item}>
+                <Link to="/wishlist">Products</Link>
```

Comment:
I think the route here is a typo,

Reply:
I didnt have the route. I have changed it now.